Tests/UI/New UI/BillingForm/ViewModel/PaymentViewModelTests.swift

```diff
@@ -91,4 +91,31 @@ class PaymentViewModelTests: XCTestCase {
         let expectedSummaryText = try XCTUnwrap(viewModel.paymentFormStyle?.editBillingSummary?.summary?.text)
         XCTAssertEqual(expectedSummaryText, summaryValue)
     }
+
+  func testSummaryTextWithEmptyValue() throws {
+      let userName = "User"
+
+    let address = Address(addressLine1: "Checkout.com",
+                          addressLine2: "",
+                          city: "London city",
+                          state: "London state",
+                          zip: "N12345",
+                          country: Country(iso3166Alpha2: "GB"))
+
+      let phone = Phone(number: "077 1234 1234",
+                        country: Country(iso3166Alpha2: "GB"))
+      let billingFormData = BillingForm(name: userName,
+                                        address: address,
+                                        phone: phone)
+      viewModel = DefaultPaymentViewModel(cardValidator: CardValidator(environment: .sandbox),
+                                          billingFormData: billingFormData,
+                                          paymentFormStyle: DefaultPaymentFormStyle(),
+                                          billingFormStyle: DefaultBillingFormStyle(),
+                                          supportedSchemes: [.unknown])
+
+      let summaryValue = "User\n\nCheckout.com\n\nLondon city\n\nLondon state\n\nN12345\n\nUnited Kingdom\n\n077 1234 1234"
```

Comment:
what we are testing is 2 spaces ? here \n\n

Reply:
testing empty value for addressline2